releasetasks/templates/desktop/publish_balrog.yml.tmpl

```diff
@@ -1,5 +1,5 @@
+{% if not release_eta %}
```

Comment:
Is there anything outside of the task graph that watches for completion of these builders? If so, they may be confused when the current builder name doesn't exist for most graphs.